packages/docs/concepts/overview.yaml

```diff
@@ -24,22 +24,23 @@ _ref:
         type: Markdown
         properties:
           content: |
-            ##### TODO: This page needs to be updated.
+            A Lowdefy app is simply a [Next.js](https://nextjs.org) application, that interprets / evaluates Lowdefy config. Lowdefy apps are written as YAML configuration files, which is easy to read, write and understand - and can be managed in source control. This makes Lowdefy quick to learn, maintain or pick up where others left off.
```

Comment:
...which  ~is~ **are** easy to read, write and understand...

Reply:
Change made in d7f5f958bb5326e80b79cad66c7c811233af2893